### ISSUE TITLE
NEWS and README not updated

### ISSUE THREAD

**Issue by msberends** [metacran/metacranweb#113]
We see that https://www.r-pkg.org/pkg/AMR shows version 0.5.0 on both the README and NEWS, which are actually the files from 2 years ago. The manual button loads the last version well.

Aren't the README and NEWS read in periodically from https://cran.r-project.org/web/packages/AMR/readme/README.html or https://github.com/cran/AMR? Not sure if this is also the case with other packages.

**Comment by gustavdelius** [comment]
Yes, this problem is also present for other packages. For example https://www.r-pkg.org/pkg/mizer still shows Readme and News from version 1.0.1 from 2019. Everything else is from the latest version 2.1.0

**Comment by msberends** [comment]
Any news? @gaborcsardi 

Our README on METACRAN now has a delay of almost 3 years. This issue is very confusing for users, especially since we focus on non-dev users.
